index.html

```diff
@@ -75,33 +87,146 @@ <h1 class="title">
       <h2 class="my-recent">My Recent Works</h2>
-      <hr />
-      <div class="placeholder">
-        <img src="images/Img Placeholder.svg" alt="Image Placeholder" />
+      <hr class="hori-break1" />
+      <div class="clearfix">
+        <div class="placeholder">
+          <img src="images/Img Placeholder.svg" alt="Image Placeholder" />
+        </div>
+        <img
+          class="placeholder2"
+          src="images/Img Placeholder-desktop.svg"
+          alt=""
+        />
+        <div class="stories">
+          <p>Multi-Post Stories</p>
+        </div>
+        <p class="daily">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="lruby">
+          <a class="block" href="#">
+            <li>css</li>
+          </a>
+          <a class="block" href="#">
+            <li>html</li>
+          </a>
+          <a class="block" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <a class="see-project" href="#"> See Project </a>
       </div>
-      <div class="stories">
-        <p>Multi-Post Stories</p>
+    </section>
+
+    <section class="project-cards-desktop">
+      <div class="nothing">
+        <button class="project-button-desktop" href="#">See Project</button>
+      </div>
+      <div class="profesion-desktop">
+        <h2 class="heading-content-desktop">
+          Data Dashboard <br />
+          Healthcare
+        </h2>
+        <p class="p-content-desktop">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language-desktop">
+          <a class="block-two-desktop" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+      </div>
+      <div class="profesion-desktop3">
+        <h2 class="heading-content-desktop3">Website Portfolio <br /></h2>
+        <p class="p-content-desktop">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language-desktop">
+          <a class="block-two-desktop" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two-desktop" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+      </div>
+      <div class="profesion-desktop4">
+        <h2 class="heading-content-desktop4">
```

Comment:
- [ ] Your see project is not visible, Kindly fix it 👍🏽 

| Figma Design | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/193132983-1c30022f-4242-4c00-95ac-d6f6d67b25c8.png) | ![image](https://user-images.githubusercontent.com/63473584/193133286-f796c556-e03d-4b37-9340-b14890a75642.png)  |